src/api/c/quiz3/notes.txt

```diff
@@ -1,3 +1,4 @@
 Compiles in:
```

Comment:
Need to move it to a section in `doc/CSI230F17.md` file with more details on tests, limitations and instructions for this specific driver.